### PR TITLE
added Open Graph support

### PR DESCRIPTION
**Pull request by t04glovern**
- Added open graph support
- added new image for Matt's post (feel free to change it)

### CODE REVIEW

**Review by sbracegirdle — APPROVED** [review]

File: _includes/head.html [hunk 1]
```diff
@@ -6,6 +6,13 @@
   <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
   <meta name="description" content="{{ page.excerpt | default: site.description | strip_html | normalize_whitespace | truncate: 160 | escape }}">
 
+  <meta property="og:type" content="article">
+  <meta property="og:title" content="{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}">
+  <meta property="og:description" content="{{ page.excerpt | default: site.description | strip_html | normalize_whitespace | truncate: 160 | escape }}">
+  <meta property="og:url" content="{{ site.url }}{{ page.url }}">
+  <meta property="og:site_name" content="{{ site.title }}">
+  {% if page.image %}<meta property="og:image" content="{{ site.url }}/{{ page.image }}">{% endif %}
```
Comment (sbracegirdle) [inline _includes/head.html:14]: Are all `page.image` relative paths?

Comment (sbracegirdle) [reply _includes/head.html:14]: I randomly checked about 30 and they all seemed to be

Comment (t04glovern) [reply _includes/head.html:14]: They look like this

```
image: img/bug_with_lens.png
```

However I just noticed there's a couple posts with another format so I'll change those too

Comment (t04glovern) [reply _includes/head.html:14]: Updated the troublesome posts here: [`cbb031a` (#128)](https://github.com/MechanicalRock/mechanicalrock.github.io/pull/128/commits/cbb031a5d86e13174bbefac75add27453bce11a3)

**Review by jgunnink — APPROVED** [review]

File: _posts/2020-01-02-codebuild-reports.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title:  "AWS CodeBuild Reports"
 date: 2020-01-02
 tags: aws codebuild reports testing devops
 author: JK Gunnink
-image: /img/blog/codebuild-reports/.png
+image: img/blog/codebuild-reports/.png
```
Comment (jgunnink) [inline _posts/2020-01-02-codebuild-reports.md:7]: Woops.

**Review by timveletta — APPROVED** [review]

**Review by matt-tyler — APPROVED** [review]

**Pull request merged**